back/dora/core/tests/test_throttling.py

```diff
@@ -0,0 +1,120 @@
+from django.core.cache import cache
+from django.test import override_settings
+from django.urls import path
+from rest_framework.response import Response
+from rest_framework.test import APITestCase
+from rest_framework.views import APIView
+
+from dora.core.test_utils import make_user
+from dora.core.throttling import ClientIPAnonRateThrottle
+
+ANON_THROTTLE_RATE_PER_MINUTE = 24
+
+
+class ThrottleTestView(APIView):
+    permission_classes = []
+
+    throttle_classes = [
+        ClientIPAnonRateThrottle,
+    ]
+
+    def get(self, request):
+        return Response({"message": "success"})
+
+
+urlpatterns = [
+    path("test-throttle/", ThrottleTestView.as_view()),
+]
+
+
+@override_settings(ROOT_URLCONF=__name__)
```

Comment:
Ici j'utiliserais plutôt le _marker_ fait pour ça : https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest-mark-urls-override-the-urlconf

Et vu que le comportement est pris en charge par DRF alors je pense qu'on a pas besoin de retester le comportement de la lib et donc avoir au final seulement 1 test qui vérifie que c'est bien activé et configuré chez nous en appelant la vue la plus simple (`dora.core.views.ping` ?), ce qui permet de ne pas avoir à créer une vue et tout le tintouin juste pour les tests.